src/awkward/_v2/_connect/jax/__init__.py

```diff
@@ -1 +1,33 @@
 # BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+
+try:
+    import jax
+
+    error_message = None
+
+except ModuleNotFoundError:
+    jax = None
+    error_message = """to use {0}, you must install jax:
+
+    pip install jax jaxlib
+
+or
+
+    conda install -c conda-forge jax jaxlib
+"""
+
+
+def import_jax(name="Awkward Arrays with JAX"):
+    if jax is None:
+        raise ImportError(error_message.format(name))
+    return jax
+
+
+def register_jax():
+    import awkward as ak
+    from awkward._v2._connect.jax.nplike import Jax
+
+    # JAX Specific Configurations
+    jax.config.update("jax_enable_x64", True)
+
+    ak._v2._util._backends["jax"] = Jax
```

Comment:
It would be better to not need a `register` function. (The `ak._v2.numba.register()` is temporary; it exists because v1 is already registered so v2 has to be opt-in.)

This particular function doesn't do anything that seems to be necessary. Awkward should not be setting any global JAX settings. If the tests need that, then the tests should call `jax.config.update("jax_enable_x64", True)` directly—it shouldn't be hiding inside this innocent-looking function.

Is there a reason that `ak._v2._util._backends` can't be initialized with `Jax` in it? Couldn't it be made possible to declare the `Jax` class without importing `jax`/`jaxlib`? (It's possible for `Cupy`.) What is the `ak._v2._util._backends` needed for, anyway?

Reply:
Yeah, we don't need a `register` function. I removed that. The `ak._v2._util._backends` is needed to map a user-level name to an nplike so that in the future if any of the backend names need to change we just change the name in the dict and we won't have to touch anything else in the codebase :)